pkg/cmdhelpers/addon_helpers.go

```diff
@@ -1,65 +1,77 @@
-package addons
+package cmdhelpers
```

Comment:
the cmdhelpers package will exist for this and the Workflow CLI functionality for now.  We will plan and migrate this logic somewhere more beneficial to templates/cli integration later on